src/darwin/Framework/CHIP/MTRDevice.mm

```diff
@@ -2523,8 +2502,6 @@ - (void)_setupSubscriptionWithReason:(NSString *)reason
                                return;
                            }
 
-                           MTR_LOG("%@ Subscribe with data version list size %lu, reduced by %lu", self, static_cast<unsigned long>(dataVersions.count), static_cast<unsigned long>(dataVersionFilterListSizeReduction));
```

Comment:
Actually, could we keep this log, but remove the `dataVersionFilterListSizeReduction` bits? This is the only place that MTRDevice logs when it starts a subscription.

Reply:
Yes, can do.

Reply:
Done.